daprdocs/content/en/developing-applications/building-blocks/state-management/howto-share-state.md

```diff
@@ -33,7 +33,6 @@ apiVersion: dapr.io/v1alpha1
 kind: Component
 metadata:
   name: statestore
-  namespace: production
```

Comment:
Need to revert this one, since production namespace is used in the example description